.gitignore

```diff
@@ -12,6 +12,9 @@
 www/assets/bundles/*
 www/webpack-stats.json
 
+# Remove potentially large or licensed content
+*.pdf
```

Comment:
Change to `textbook-tool/*.pdf` allowing for the inclusion `.pdf`s for demo testing data elsewhere in the app.